Makefile

```diff
@@ -34,36 +12,36 @@ bash: run clean
 dev: run
 
 .PHONY: run
-run: compose
+run:
 	$(COMPOSE) run --rm test /bin/bash
 
 .PHONY: run_test
-run_test: compose
+run_test:
 	$(COMPOSE) run --rm -e COVERAGE=$(COVERAGE) test
```

Comment:
I think we should get rid of the `pisoni` image and run the next command here:

```
THREESCALE_CORE_INTERNAL_API=http://user:password@apisonator:3000 bundle exec rake test
```

Then create a new make target to launch apisonator (from podman-compose) and make this target depend on that one.

Reply:
can you run compose in circleci?

Reply:
> can you run compose in circleci?

Yes, this is how it used to work before, I think because there was a specific `pisoni-ci` image, that probably included `docker-compose`. And there was also a fallback way that downloaded the `docker-compose` library.

Reply:
I doubt circleci supported container in container. Maybe previously it ran on a VM that started containers?